src/App.tsx

```diff
@@ -1,61 +1,41 @@
 import './App.scss';
+import { TodoList } from './components/TodoList';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import React, { useState } from 'react';
+import { Todo } from './types/Todo';
+import { AddTodoForm } from './components/AddTodoForm/AddTodoForm';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+export const getUsersById = (id: number) =>
+  usersFromServer.find(user => user.id === id);
```

Comment:
I'd recommend creating a separate file for helper functions, especially if you're exporting them somewhere

Reply:
good point 🔥 
move it to a separate directory - `utils`